### ISSUE TITLE
A Ton of Glares

### ISSUE THREAD

**Issue by Cicopath** [Faboslav/friends-and-foes#84]
### Minecraft version information

1.19.2

### Mod loader information

Forge

### Mod loader version information

43.2.0

### Mod version information

1.6.3

### Expected Behavior

I expect Glares to not spawn in my basement. I'm in an Orchard biome, and the area where my axolotls are on the right, a few blocks become a Dripstone Cave biome.

### Actual Behavior

Glares are spawning in my basement. Slowly, more and more spawn. I have about seven or eight total inside my house now.

### Reproduction Steps

1. Have my exact setup at Y-level 58.
2. Just wait for Glares to spawn.

### Screenshots & files

![2022-12-28_21 39 54](https://user-images.githubusercontent.com/61031176/209896386-96dd240d-94da-4a4c-8eb7-4e87a00f47b2.png)
![2022-12-28_21 42 02](https://user-images.githubusercontent.com/61031176/209896388-1a5e9bb0-ac20-421a-9fca-70a3cce0a4c5.png)

**Comment by Faboslav** [comment]
This is not a bug, `orchard` biome is technically marked as lush biome under the custom forge tag `#forge:is_lush`, so simply said it belongs to the "lush biome category". Also all other spawn conditions are met, which is:
- block is below Y level 63
- sky is not visible
- light level is more than 5
- and spawn block is one of these:
```
"minecraft:moss_block",
"minecraft:moss_carpet",
"minecraft:azalea",
"minecraft:flowering_azalea",
"minecraft:grass",
"minecraft:small_dripleaf",
"minecraft:big_dripleaf",
"minecraft:clay" 
```

**Comment by Faboslav** [comment]
Although, i should probably stop using that tag, as it is just a naming coincidence and it should not be related to the "lush caves", i will tweak this with the next update :).

**Comment by Cicopath** [comment]
Oh, I didn't even know the Orchard biome has that tag! And I'm glad that a maximum of only 15 can spawn, I don't need a house inundated with Glares lol. But I certainly agree, it doesn't relate to the Lush Caves at all. I know there are a few biomes similar to the Lush Caves, such as the Lush Stacks from Oh The Biomes You'll Go, though it's a surface biome; and the Glowing Grotto from Biomes O' Plenty with blue Glowing Moss blocks.

Now that I think about it, a blue Glare exclusive to the Glowing Grotto sounds cool!

**Comment by Faboslav** [comment]
I think hopefully i will be able to release the update today :).

**Comment by Cicopath** [comment]
Take your time!

**Comment by Faboslav** [comment]
Should be fixed with 1.6.7
